texar/modules/regressors/regressor_base.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2019 The Texar Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Base class for Regressors.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from texar.module_base import ModuleBase
+
+__all__ = [
+    "RegressorBase"
+]
+
+
+class RegressorBase(ModuleBase):
+    """Base class inherited by all regressor classes.
+    """
+
+    def __init__(self, hparams=None):
+        ModuleBase.__init__(self, hparams)
+
+    @staticmethod
+    def default_hparams():
+        """Returns a dictionary of hyperparameters with default values.
+        """
+        return {
+            "name": "classifier"
+        }
+
+    def _build(self, inputs, *args, **kwargs):
+        """Runs regressors on inputs.
+
+        Args:
+          inputs: Inputs to the classifier.
```

Comment:
"classifier" --> "regressor"

Reply:
Done! [9280366]

Reply:
hmm.. I still see "classifier" here

Reply:
Interesting. https://github.com/AvinashBukkittu/texar/blob/xlnet/texar/modules/regressors/regressor_base.py#L41 I can see the change here.